### PR TITLE
chore(PPDSC-2725): banner

### PR DESCRIPTION
**Pull request by mstuartf**
[PPDSC-2725](https://nidigitalsolutions.jira.com/browse/PPDSC-2725)

**What**

1. Background - why this is needed
2. What did you do
Tested original styling can be restored:
Screenshot 2023-02-06 at 16 33 51
Screenshot 2023-02-06 at 16 34 07
Updated Storybook styling so button is full width:
Screenshot 2023-02-06 at 16 33 51

3. What does the reviewers should expect

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

[PPDSC-2725]: https://nidigitalsolutions.jira.com/browse/PPDSC-2725?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by pp-serviceaccount** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2725-banner
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2725-banner/storybook

File: src/banner/types.ts [hunk 1]
```diff
@@ -14,10 +14,6 @@ export interface BannerBaseProps extends React.AriaAttributes {
   overrides?: {
     stylePreset?: MQ<string>;
     minHeight?: MQ<string>;
-    /**
-     * @deprecated This property is deprecated and will be removed in the next major release. Use `paddingInline` & `paddingBlock` instead.
-     */
-    spaceInset?: MQ<string>;
```
Comment (mutebg) [inline src/banner/types.ts:20]: Maybe I've missed something, did we decide to start removing spaceInset and replace it with logical props?

Comment (mstuartf) [reply src/banner/types.ts:20]: No be fair I just assumed because it says that it `will be removed in the next major release`. 

Shall I keep it?

Comment (mstuartf) [reply src/banner/types.ts:20]: On reflection it makes sense to keep it. I'll make the change.

**Review by JohnTParsons — APPROVED** [review]

**Pull request merged**